GameDataParser/Parsers/ColorPaletteParser.cs

```diff
@@ -48,11 +48,15 @@ protected override List<ColorPaletteMetadata> Parse()
                             byte[] tertiaryBytes = BitConverter.GetBytes(tertiary);
                             Array.Reverse(tertiaryBytes);
 
-                            EquipColor newColor = EquipColor.Custom(
+                            int paletteColor = Convert.ToInt32(colorNode.Attributes["palette"].Value, 16);
+                            byte[] paletteBytes = BitConverter.GetBytes(paletteColor);
+                            Array.Reverse(paletteBytes);
+
+                            MixedColor newColor = MixedColor.Custom(
                     Color.Argb(primaryBytes[0], primaryBytes[1], primaryBytes[2], primaryBytes[3]),
```

Comment:
Optional but could be cool to have a method called FromBytes in Color that just takes in bytes and does this whole index thing there